examples/nvflare_setup.ipynb

```diff
@@ -16,7 +21,12 @@
    "id": "64ba2b87-d3be-4058-8156-9ef589bce9cf",
    "metadata": {},
    "source": [
-    "### Install NVFLARE from PyPI\n"
+    "### Install NVFLARE from PyPI\n",
+    "```\n",
+    "pip install 'nvflare>=2.3.0rc5'\n",
+    "\n",
+    "```\n",
+    "if you only interested in python module ( not CLI commands), the following you can do in notebook\n"
```

Comment:
suggests change the text to:

"If you want to use NVFlare CLI commands in the terminal, we recommend to run pip install directly in the terminal."
"We don't recommend running NVFlare CLI commands in jupyter notebook cells."